Applications/Spire/Source/TimeAndSales/TimeAndSalesWindow.cpp

```diff
@@ -1,40 +1,183 @@
 #include "Spire/TimeAndSales/TimeAndSalesWindow.hpp"
+#include "Spire/TimeAndSales/NoneTimeAndSalesModel.hpp"
```

Comment:
Include in alphabetical order.

Reply:
I don't see the order issue. Since this is the source file of `TimeAndSalesWindow`, `TimeAndSalesWindow.hpp` should be placed first.

Reply:
Sorry. I see what the issue is.